museum/sql/testImports/baseObject.sql

```diff
@@ -0,0 +1,16 @@
+# find at least 2 representative values for each field
+(SELECT PrimaryKey_Object_Table FROM Objects WHERE ItemType IS NOT NULL LIMIT 2)
```

Comment:
Should these two queries be combined?

    SELECT <fields>
    FROM Objects
    WHERE PrimaryKey_Object_Table IN (
        (SELECT PrimaryKey_Object_Table FROM Objects WHERE ItemType IS NOT NULL LIMIT 2)
        UNION <etc>
    )

?